test/test.py

```diff
@@ -0,0 +1,98 @@
+import munge
+import unittest
+
+
+def file_factory(n):
+    # makes a file with "the great big whale splashed the timid blue fox.\n" repeated n times
+    with open("test_files/simple_whale_" + str(n) + ".txt", "w") as out:
```

Comment:
Usually it's not *great* practice to write new files during test, but in this case it's necessary...but let's think about how to do this politely (e.g. by making sure the directory you're writing to exists, removing the file after we're done, etc.)

`TestCase` comes with two methods, `setUp()` and `tearDown()`, with the former running before all test cases and the latter running after all test cases. I would recommend overriding these to create and delete your test files. You could construct your test file in the `setUp` and remove it in `tearDown` so you can recycle files in your test without having your test cases be responsible for writing and deleting (after all, if one fails, it might not run all the necessary steps!).